pkgs/stdenv/adapters.nix

```diff
@@ -335,6 +335,22 @@ rec {
             }
       );
 
+  useWildLinker =
+    stdenv:
+    if !stdenv.targetPlatform.isLinux then
+      throw "Wild only supports Linux hosts"
```

Comment:
"hosts" here could be misleading. I am not sure what it means either! Can it do non-linux ELFs? Can it run on other systems and cross-link?

Reply:
It can only run on Linux and it can only target linux.

Reply:
Target sucks